osu.Game.Rulesets.Catch/Objects/JuiceStream.cs

```diff
@@ -41,6 +41,12 @@ public double SliderVelocityMultiplier
             set => SliderVelocityMultiplierBindable.Value = value;
         }
 
+        /// <summary>
+        /// An extra multiplier that affects the number of <see cref="Droplet"/>s generated by this <see cref="JuiceStream"/>.
+        /// An increase in this value increases <see cref="TickDistance"/>, which reduces the number of ticks generated.
+        /// </summary>
+        public double TickDistanceMultiplier = 1;
```

Comment:
Is it worth noting in the xmldoc that this is used for legacy purposes? Or are you thinking this something we might use in the future?

Reply:
I copied this xmldoc straight from the osu! ruleset.

Reply:
Hmm okay.